docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -695,7 +714,7 @@ NOTE: `MYPAS_WORD` is a sample password and should be never used, own strong pas
 http.auth.password=ENC(WVnXhzG704KRZZFYL7hfpA4YoSm8V2F5eNXyr7CRYCichteHvNS3nB23zaI8CuLy)
 ----
 
-Please see <<_externalized_configuration>> to find the ways how encryptor password can be passed to the tests.
+Please see <<_externalized_configuration>> to get more information about how encryptor password can be passed to the tests.
```

Comment:
any particular reason for this change? I'm not against, but trying to understand the nature

Reply:
> any particular reason for this change? I'm not against, but trying to understand the nature

Above I described three ways to configure a password for encryption
For some users this may be more than enough
But for someone it will be useful to read about these configuration methods not in the context of setting a password for the encryptor, but "globally"